### PR TITLE
GitHub logo

### PR DESCRIPTION
**Pull request by RDiMag**

### CODE REVIEW

File: src/pages/SignUp/SignUp.tsx [hunk 1]
```diff
@@ -96,6 +100,7 @@ const SignUp = () => {
         </AuthForm>
         <AuthSub>Already have an account?</AuthSub>
         <AuthRedirectLink to="/sign-in">Login Here</AuthRedirectLink>
+        <CookieRequest>Please make sure cookies are enabled.</CookieRequest>
```
Comment (theodorusandi) [inline src/pages/SignUp/SignUp.tsx:103]: alert

What do you think if we have something like this on top of the sign in card?

Comment (RDiMag) [reply src/pages/SignUp/SignUp.tsx:103]: I like that! Should it be a static alert that is always visible? 

I can do something like this using one of the colors from our theme file.

File: src/pages/SignIn/SignIn.tsx [hunk 2]
```diff
@@ -57,7 +58,10 @@ const SignIn = () => {
     <AuthContainer>
       <AuthCard>
         <AuthHead>Sign In</AuthHead>
-        <TwitterIcon />
+        <a href="http://localhost:5000/auth/github">
+          {' '}
```
Comment (jpatrici) [inline src/pages/SignIn/SignIn.tsx:62]: do we need this space?

File: src/pages/SignUp/SignUp.tsx [hunk 3]
```diff
@@ -57,7 +58,10 @@ const SignUp = () => {
     <AuthContainer>
       <AuthCard>
         <AuthHead>Sign Up</AuthHead>
-        <TwitterIcon />
+        <a href="http://localhost:5000/auth/github">
```
[the remark this answers is not in the document]
Comment (jpatrici) [reply src/pages/SignUp/SignUp.tsx:61]: you can reuse `GITHUB_AUTH_URL` from my other comment

File: src/pages/SignIn/SignIn.tsx (hunk 2 above)
Comment (RDiMag) [reply src/pages/SignIn/SignIn.tsx:62]: I don't think so, I deleted it.

File: src/pages/SignIn/SignIn.tsx [hunk 4]
```diff
@@ -57,7 +58,10 @@ const SignIn = () => {
     <AuthContainer>
       <AuthCard>
         <AuthHead>Sign In</AuthHead>
-        <TwitterIcon />
+        <a href="http://localhost:5000/auth/github">
```
[the remark this answers is not in the document]
Comment (RDiMag) [reply src/pages/SignIn/SignIn.tsx:61]: I've done this in both Sign In and Sign Up pages and it isn't connecting to the GitHub authorization. Maybe we can pair Monday?

File: .env [hunk 5]
```diff
@@ -1 +0,0 @@
-REACT_APP_BASE_URL="http://localhost:5000"
```
Comment (jpatrici) [inline .env:1]: hmm I thought this was deleted in #83 ? 🤔

Comment (jpatrici) [reply .env:1]: ohh i just realized you haven't updated the branch with main

**Comment by jpatrici** [comment]
@RDiMag do you want to update this PR with the main branch to get the latest update on the auth styling? Otherwise the code looks good and ready for approval! just want to make sure they all still look okay after the rebase

**Review by jpatrici — APPROVED** [review]

**Pull request merged**